FreeRTOS_DHCP.c

```diff
@@ -1449,8 +1451,10 @@
                  * returned to the stack. */
                 FreeRTOS_ReleaseUDPPayloadBuffer( pucUDPPayloadBuffer );
             }
-
-            xResult = pdTRUE;
+            else
```

Comment:
Is there a unit test for this code?  If so, can it be updated to cover this condition?

Reply:
There are no unit-tests as of now. We are working on adding unit-tests to get 100% coverage.